coderd/notifications/enqueuer.go

```diff
@@ -26,6 +26,12 @@ var (
 	ErrDuplicate                         = xerrors.New("duplicate notification")
 )
 
+// IsSeriousEnqueueError filters out trivial errors which can be ignored by
```

Comment:
Did you consider making `StoreEnqueuer.EnqueueWithData` suppress the error, and possibly logging this case itself under debug level? Debouncing this might also be a good idea if perhaps a little overengineered.

I don't think the caller cares that the notification is disabled, and you're right that this is superfluous.

Reply:
Cool, we could then check in those tests for the presence of that log message then. That should satisfy the intent of the tests; the fact that it's an error right now is not really relevant.

I mainly want to avoid having folks needing to remember to use `IsSeriousEnqueueError` at all call sites; that won't last long.